apps/dashboard/src/components/inbox-button.tsx

```diff
@@ -108,7 +110,7 @@ export const InboxButton = () => {
 
   return (
     <Inbox
-      subscriberId={user.externalId ?? ''}
+      subscriberId={`org_${currentOrganization?._id ?? ''}:user_${user.externalId ?? ''}`}
```

Comment:
```suggestion
      subscriberId={`org_${currentOrganization._id}:user_${user.externalId}`}
```

you can add the currentOrganization to the `if` statement on line 93. we cannot generate a subscriber if it doesn't have the proper values.